schema.json

```diff
@@ -0,0 +1,214 @@
+{
+  "id": "http://api.codeclimate.com/test-coverage-schema#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+
+  "title": "Test Coverage",
+  "description": "Schema that describes the format of the test coverage payload sent to Code Climate",
+
+  "definitions": {
+    "covered_percent": {
+      "type": "integer",
+      "minimum": 0,
+      "maximum": 100,
+    },
+    "covered_strength": {
+      "type": "integer",
+      "minimum": 0,
+    },
+
+    "line_counts": {
+      "type": "object",
+      "properties": {
+        "covered": {
+          "description": "Number of covered lines of code",
+          "type": "integer",
+          "minimum": 0,
+        },
+        "missed": {
+          "description": "Number of uncovered lines of code",
+          "type": "integer",
+          "minimum": 0,
+        },
+        "total": {
+          "description": "Total lines of code",
+          "type": "integer",
+          "minimum": 0,
+        },
+      },
+      "required": ["covered", "missed", "total"],
+    },
+
+    "git_sha": {
+      "type": "string",
+      "pattern": "^\d{1,}\.\d{1,}\.\d{1,}",
```

Comment:
Ooops. C&P error I think. I originally had the pattern as `/^[a-zA-Z0-9]{40}$/` .. does that work or is it too specific to SHA-1? If so, what pattern, if any, do you recommend?